subsys/bluetooth/mesh/brg_cfg.c

```diff
@@ -304,4 +314,6 @@ void bt_mesh_brg_cfg_tbl_remove(uint16_t net_idx1, uint16_t net_idx2, uint16_t a
 		bt_mesh_settings_store_schedule(BT_MESH_SETTINGS_BRG_PENDING);
```

Comment:
No, this will not work. Because then `store` will have to be declared first, and then perhaps we hope that compiler removes it. But side effect will be `store = true` few lines above will have to be kept, which will consume extra code. So I don't think it is worth simplifying this.